engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -806,6 +807,7 @@ namespace dmGameSystem
     {
         GuiComponent* gui_component = (GuiComponent*)*params.m_UserData;
         dmGui::Result result = dmGui::FinalScene(gui_component->m_Scene);
+        dmGui::DeleteDynamicTextures(gui_component->m_Scene, &DeleteTexture);
```

Comment:
After "Final" I expect the scene to be invalid (i.e. deallocated).

And I think deleting of the textures should be handled by the FinalScene.

Since the pattern is that dmGui takes some Params struct, I suggest it is the pattern we should use here as well.